server/service/src/invoice/outbound_return/delete/mod.rs

```diff
@@ -0,0 +1,211 @@
+use repository::{ActivityLogType, InvoiceRowRepository, RepositoryError, TransactionError};
+
+pub mod validate;
+
+use validate::validate;
+
+use crate::{
+    activity_log::activity_log_entry,
+    invoice::common::get_lines_for_invoice,
+    invoice_line::stock_out_line::{
+        delete::{delete_stock_out_line, DeleteStockOutLine, DeleteStockOutLineError},
+        StockOutType,
+    },
+    service_provider::ServiceContext,
+    WithDBError,
+};
+
+type OutError = DeleteOutboundReturnError;
```

Comment:
Why not just use `DeleteOutboundReturnError` directly?
I guess this saves typing but personally I prefer to see the type...